### PR TITLE
UX Improvements for Distribution

### PR DESCRIPTION
**Pull request by ezequielc**
Updated wording
Added call to action to Select Displays in the right side
Added a flag to hide CTA (needed in Placeholder Settings because it was using too much space)
 
@alex-deaconu Please review. Thanks

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Now that you are removing the Add Display state in the other PR, any suggestion on how I could call the modal from the component? Thanks

**Comment by alex-deaconu** [comment]
@ezequielc good question. My suggestion is to emit an event, which will trigger the modal in the displayFactory. Thoughts? Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Yes, that is what I thought as well. I will update the code once you merge the other branch to master. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Please review the additional commit. Thanks

File: src/distribution-selector/ctr-distribution-list.js [hunk 1]
```diff
@@ -20,6 +17,11 @@ angular.module("risevision.common.components.distribution-selector")
         id: "displaySearchInput"
       };
 
+      $scope.$on("displayCreated", function () {
+        $scope.displays = new BaseList(DB_MAX_COUNT);
```
Comment (alex-deaconu) [inline src/distribution-selector/ctr-distribution-list.js]: @ezequielc wondering why not use `$scope.displays.clear();` (see `doSearch`)? Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Just one suggestion re: consistency. Thanks.

File: src/distribution-selector/ctr-distribution-list.js (hunk 1 above)
Comment (ezequielc) [inline src/distribution-selector/ctr-distribution-list.js]: @alex-deaconu I missed that, will update. Thanks

File: dist/js/distribution-selector.js [hunk 2]
```diff
@@ -159,6 +156,11 @@ angular.module("risevision.common.components.distribution-selector")
         id: "displaySearchInput"
       };
 
+      $scope.$on("displayCreated", function () {
+        $scope.displays = new BaseList(DB_MAX_COUNT);
+        $scope.load();
+      });
```
Comment (alex-deaconu) [inline dist/js/distribution-selector.js]: @ezequielc just a heads up, this is missing build results. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Code was updated as suggested. Please review. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍

**Pull request merged**